pkg/controller/component/lifecycle/lfa_component.go

```diff
@@ -17,37 +17,42 @@ You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-package handlers
+package lifecycle
 
 import (
 	"context"
 
-	"github.com/go-logr/logr"
-	"github.com/pkg/errors"
-	ctrl "sigs.k8s.io/controller-runtime"
-
-	"github.com/apecloud/kubeblocks/pkg/kb_agent/util"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-type GRPCHandler struct {
-	Logger logr.Logger
+type postProvision struct {
+	namespace   string
+	clusterName string
+	compName    string
+}
+
+var _ lifecycleAction = &postProvision{}
+
+func (a *postProvision) name() string {
+	return "postProvision"
 }
 
-var _ Handler = &GRPCHandler{}
+func (a *postProvision) parameters(ctx context.Context, cli client.Reader) (map[string]string, error) {
+	return nil, nil
```

Comment:
It seems "$KB_CLUSTER_COMPONENT_DELETING_LIST" and "$KB_CLUSTER_COMPONENT_UNDELETED_LIST" are used in redis addon's postProvision and preTerminate.

@Y-Rookie I remember you said these envs are used to determining which shards are being deleted by scale-in when handling scale-out. Do you need them any longer?

Reply:
Both actions are executed through jobs instead of lorry, so this PR does not provide support for them through kb-agent.

Reply:
> It seems "$KB_CLUSTER_COMPONENT_DELETING_LIST" and "$KB_CLUSTER_COMPONENT_UNDELETED_LIST" are used in redis addon's postProvision and preTerminate.
> 
> @Y-Rookie I remember you said these envs are used to determining which shards are being deleted by scale-in when handling scale-out. Do you need them any longer?

This PR hasn't switched postProvision and preTerminate yet, so it's not affected at the moment. Later, when migrating to kb-agent, we can take a look at how to replace these environment variables.